troubleshooting/state-migration/exercise/build.gradle

```diff
@@ -17,21 +17,21 @@
 
 buildscript {
     repositories {
-        jcenter()
+        gradlePluginPortal()
+        mavenCentral()
     }
     dependencies {
-        classpath "com.commercehub.gradle.plugin:gradle-avro-plugin:0.16.0"
+        classpath "com.github.davidmc24.gradle.plugin:gradle-avro-plugin:1.9.1"
     }
 }
-apply plugin: "com.commercehub.gradle.plugin.avro"
+apply plugin: "com.github.davidmc24.gradle.plugin.avro"
 apply plugin: 'application'
 
 ext.javaExerciseClassName = 'com.ververica.flink.training.exercises.avro.StateMigrationJob'
 
 mainClassName = ext.javaExerciseClassName
 
 dependencies {
-    implementation "org.apache.flink:flink-avro:${flinkVersion}"
-
+    implementation "org.apache.avro:avro:1.9.1"
```

Comment:
I am not sure whether we can remove flink-avro...

Reply:
good catch. Fixed